tests/processor/test_utils.py

```diff
@@ -32,80 +37,47 @@ def test_download_build(m_popen, m_get_koji_session, m_assert_command):
     m_koji_session = mock.Mock()
     m_koji_session.getBuild.return_value = {
         'id': 12345,
-        'task_id': 23456
     }
-    m_koji_session.getTaskInfo.return_value = {'method': 'build'}
+
     m_get_koji_session.return_value = m_koji_session
-    m_process = mock.Mock()
+    error = b''
+
+    m_process_rpm = mock.Mock()
     output = (b'Downloading: resultsdb-2.1.0-2.el7.noarch.rpm\n'
               b'[===========================         ]  75%  64.00 KiB\r'
               b'[====================================] 100%  84.34 KiB\r\n'
               b'Downloading: resultsdb-2.1.0-2.el7.src.rpm\n'
               b'[============================        ]  80%  64.00 KiB\r'
               b'[====================================] 100%  79.61 KiB\r\n')
-    error = b''
-    m_process.communicate.return_value = (output, error)
-    m_process.returncode = 0
-    m_popen.return_value = m_process
+    m_process_rpm.communicate.return_value = (output, error)
+    m_process_rpm.returncode = 0
+
+    m_process_maven = mock.Mock()
```

Comment:
Could you make these separate tests per type? That way when the unit tests fail, we easily know which type is failing. Using `pytest.mark.parametrize` would be ideal.